projects/visual-regression/src/tsconfig.app.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Curious why this was needed? Seems like it's one folder too deep and almost matches the one already existing?

Reply:
Hmm.. I'm not sure actually. It was there in `demo/src` folder, so it was copied to visual regression project folder.

Reply:
Ohh I bet that was some cleanup we missed during The Big Migration™️